src/error.rs

```diff
@@ -13,31 +13,37 @@ use core::fmt;
 use std::{io, error};
 
 // A randomly-chosen 16-bit prefix for our codes
-pub(crate) const CODE_PREFIX: u32 = 0x57f40000;
+pub(crate) const CODE_PREFIX: u32 = 0x57f4c500;
 const CODE_UNKNOWN: u32 = CODE_PREFIX | 0;
 const CODE_UNAVAILABLE: u32 = CODE_PREFIX | 1;
 
-/// An unknown error.
-/// 
-/// This is the following constant: 57F40000 (hex) / 1475608576 (decimal).
-pub const ERROR_UNKNOWN: Error = Error(unsafe {
-    NonZeroU32::new_unchecked(CODE_UNKNOWN)
-});
-
-/// No generator is available.
-/// 
-/// This is the following constant: 57F40001 (hex) / 1475608577 (decimal).
-pub const ERROR_UNAVAILABLE: Error = Error(unsafe {
-    NonZeroU32::new_unchecked(CODE_UNAVAILABLE)
-});
-
 /// The error type.
 /// 
 /// This type is small and no-std compatible.
 #[derive(Copy, Clone, Eq, PartialEq)]
 pub struct Error(NonZeroU32);
 
 impl Error {
+    /// An unknown error.
+    /// 
+    /// This is the following constant: 57f4c500 (hex) / 1475659008 (decimal).
+    ///
+    /// The constant value is provided for reference only. It may change in
+    /// future and it will not be considered a breaking change.
```

Comment:
I think post-release we should try not to change this though?

Reply:
Yes, but I think we should specify somehow, that code should not match on the given error code and use constant instead. I guess we can change wording somehow, but I don't have good ideas right now.